WINDOWS.md

```diff
@@ -0,0 +1,165 @@
+# Setup Windows
+
+## Table of Contents
+
+1. [Setup Docker](#setup-docker)
+
+2. [Setup the application](#Setup-the-application)
+
+3. [Run the application](#Run-the-application)
+
+4. [Using the application](#using-the-application)
+
+5. [Common setup errors](#having-troubles-check-our-common-errors--gotchas)
+
+6. [Development](#development)
+
+7. [Backup and restore the Terrastories database](#backup-and-restore-the-terrastories-database)
+
+## Setup Docker
+
+Docker is a platform that serves to deliver software in packages called containers. Terrastories makes use of this to deliver the 
+many facets included in the application in one easy package. 
+
+Download Docker:
+
+- https://hub.docker.com/editions/community/docker-ce-desktop-windows/
+
+System prerequisites for Windows 10 Pro: 
+- 64 bit processor 
+- 4GB system RAM 
+- [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/). 
+
+Setup instructions for Docker on Windows 10 Pro:
+
+- https://docs.docker.com/docker-for-windows/install/
+
+System prerequisites for Windows 10 Home: 
+- Ensure that you are running on version 2004 or higher, you will need to update your system if it is running on an older version
+- Enable the WSL 2 feature on Windows, follow the instructions on this link to complete this step, this includes installing a Linux distribution such as Ubuntu
+- To complete the previous steps, you must have a 64 bit processor, 4GB system RAM, and [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/). 
+
+Setup instructions for Docker on Windows 10 Home:
+
+- https://docs.docker.com/docker-for-windows/install-windows-home/
```

Comment:
This block of information is a little hard to navigate. I think if we add some formatting, it will help future readers determine which instructions to follow:

```suggestion
| | Windows 10 Pro |  Windows 10 Home |
| --- | --- | --- |
| System Requirements | - 64 bit processor<br>- 4GB system RAM<br>- [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/) | Same as Windows 10 Pro, plus: <br>- OS Version 2004 or higher (update your system if version is older)<br>- Enable [WSL 2 feature](https://docs.docker.com/docker-for-windows/wsl/) |
| Setup Instructions | https://docs.docker.com/docker-for-windows/install/ | https://docs.docker.com/docker-for-windows/install-windows-home/ |
```

Which will look like this:

| | Windows 10 Pro |  Windows 10 Home |
| --- | --- | --- |
| System Requirements | - 64 bit processor<br>- 4GB system RAM<br>- [virtualization must be enabled](https://mashtips.com/enable-virtualization-windows-10/) | Same as Windows 10 Pro, plus: <br>- OS Version 2004 or higher (update your system if version is older)<br>- Enable [WSL 2 feature](https://docs.docker.com/docker-for-windows/wsl/) |
| Setup Instructions | https://docs.docker.com/docker-for-windows/install/ | https://docs.docker.com/docker-for-windows/install-windows-home/ |

It also allows us to add other versions of windows in a sensible manner.